README.md

```diff
@@ -1 +1,47 @@
-# todo-list
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# todo-list
+In this website you can add todo list for the day and you remove tasks that are completed.
+
+## Built With
+
+- HTML, CSS and JavaScript (ES6)
+
+## Live Demo
+
+- [Live Demo Link](https://gtekle.github.io/todo-list/)
+
+## Getting Started
+
+- Install Node.js
```

Comment:
Please add a summary to your Pull Request. It can be about project features and how you implemented them.